### PR TITLE
chore: update readme file with additional information

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
1. Add an explanation of how to run the project development
2. Remove redundant and incorrect information

## Linear Ticket
https://linear.app/autokitteh/issue/UI-695/improve-documentation-for-vscode-extension

## What type of PR is this? (check all applicable)

- [ ] 💡 (feat) - A New Feature (non-breaking change which adds functionality)
- [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
- [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
- [ ] 🏎 (perf) - Optimization
- [x] 📄 (docs) - Documentation - Documentation only changes
- [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
- [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white spaces, formatting, missing semi-colons, etc)
- [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
- [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

## Code Standards
- [ ] Notify only when user attention is needed, otherwise log to console.
  - [ ] Notifications: short description with context identifiers (e.g. id of the manipulated entity).
  - [ ] Logs: full description with full context identifiers (e.g. deploymentId and the relevant projectId).
- [ ] If you're not sure what is the proper behavior, consult with the product team.
- [ ] Reduce the use of `else` by employing early returns to make code more readable and less nested.
- [ ] MVC Separation: Ensure separation of concerns; views should only be manipulated by controllers (also the computing - sorting, fitlering, etc.), not by services, to maintain a clean MVC architecture.
- [ ] Before implementing custom logic, check if existing functions or utilities (like lodash) offer a simpler solution.
- [ ] Avoid using `!important` in CSS where possible.
- [ ] Use memoization for class calculations.
- [ ] Place constant strings in your code using I18n.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +63,9 @@ The [autokitteh](https://autokitteh.com) license identifier for this project is
 To contribute to autokitteh, please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file.
 
 We appreciate contributions from everyone!
+
+### How it built 🛠
```
Comment (daabr) [inline README.md:67]:
```suggestion
### How it's built 🛠
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,48 @@
-# 🐾 autokitteh
+# 🐾 AutoKitteh - VSCode Extension
```
Comment (daabr) [inline README.md:1]:
```suggestion
# 🐾 AutoKitteh - VS Code Extension
```

**Review by daabr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RonenMars) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (RonenMars) [reply README.md]: Changed to "Visual Studio Code Extension"

**Pull request merged**